scripts/orig_scripts/txt2img.py

```diff
@@ -198,11 +198,12 @@ def main():
         opt.ckpt = "models/ldm/text2img-large/model.ckpt"
         opt.outdir = "outputs/txt2img-samples-laion400m"
 
-    seed_everything(opt.seed)
 
     config = OmegaConf.load(f"{opt.config}")
     model = load_model_from_config(config, f"{opt.ckpt}")
 
+    seed_everything(opt.seed)
```

Comment:
Note to self: test that seeds used in previous versions generate the expected images. This shouldn't change behaviour, but just in case...